### PR TITLE
Move web content from OSCAL, and other streamlining. (Revised for Release)

### PR DESCRIPTION
**Pull request by Compton-NIST**
# Committer Notes

This is a corrected PR based on the issues around https://github.com/usnistgov/OSCAL/pull/1824.

In support of Issue #1802 

**Currently in rapid prototype mode with the team.**

- Website content moved to OSCAL-Pages project.  This project will build and publish the public website.
- Reference content moved to OSCAL-Reference project. This project will build and publish the developer documentation. AKA OSCAL Reference.
- OSCAL repo in this branch has been reduced to only the model content.

Remaining to determine:

- Linking up the websites so that it is least disruptive to the public.
- The generated xml and json content has been removed from this branch.  We need to talk about this, and what is the "right" way moving forward.  I think there are a number of options and opinions to consider.
- Figuring out the entry point into the public website. (linking, landing page, etc)

Benefits:

- Less activity on the OSCAL project unrelated to models.
- Will make it easier for the community to monitor.
- More focused workstreams between model development, website content/education and experiments with models.
- Depending on how we publish the schemas-and-converters artifact, may make it easier for the public to access and use releases of OSCAL with less tooling.
- Reference will be in position to be updated and enhanced over time for development use.
- Public website can update at-will without impacting the OSCAL model project.

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

(For reviewers: [The wiki has guidance](https://github.com/usnistgov/OSCAL/wiki/Issue-Review) on code review and overall issue review for completeness.)

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~Have you written new tests for your core changes, as applicable?~ A.J. is waving this, N/A.
- ~Have you included examples of how to use your new feature(s)?~ A.J. is waving this, N/A.
- [x] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,8 +1,3 @@
-[submodule "docs/themes/uswds"]
-	path = docs/themes/uswds
-	url = https://github.com/usnistgov/hugo-uswds.git
-	branch = master
-	update = rebase
 [submodule "build/metaschema"]
 	path = build/metaschema
 	url = https://github.com/usnistgov/metaschema.git
```
Comment (aj-stein-nist) [inline .gitmodules:3]: Maybe blocking 🤷: we still need to figure out this piece (stick with this, move to metaschema-xslt in the same PR) before we can formally, merge, right?

**Review by aj-stein-nist — COMMENTED** [review]
I am very very happy with this. I will test more but I assume we need to answer the one big question before moving forward.

File: .github/workflows/workflow-generate-metaschema-resources.yml [hunk 2]
```diff
@@ -130,24 +130,18 @@ jobs:
...
+    #   if: github.event.inputs.commit_resources == 'true' || inputs.commit_resources == true
+    #   uses: stefanzweifel/git-auto-commit-action@be7095c202abcf573b09f20541e0ee2f6a3a9d9b
+    #   with:
+    #     repository: ${{ env.CHECKOUT_PATH }}
+    #     file_pattern: xml json
+    #     skip_dirty_check: false
+    #     commit_user_name: OSCAL GitHub Actions Bot
+    #     commit_user_email: oscal@nist.gov
+    #     commit_author: OSCAL GitHub Actions Bot <oscal@nist.gov>
+    #     commit_message: Publishing generated metaschema resources [ci skip]
```
[the remark this answers is not in the document]
Comment (Compton-NIST) [reply .github/workflows/workflow-generate-metaschema-resources.yml:147]: At the time, it was not something that had been discussed widely, so I commented.  I think we will probably want to clean up/remove the status-ci-cd.yml, so we can clear it out next round.

File: .gitmodules (hunk 1 above)
Comment (Compton-NIST) [reply .gitmodules:3]: I'll take guidance here.  I'm not quite as confident on the best course.

Comment (aj-stein-nist) [reply .gitmodules:3]: @nikitawootten-nist with https://github.com/usnistgov/metaschema-xslt/pull/45 on the way and close to setting off the baking timer in the oven, I assume we are close to ready to switch out this sprint?

**Review by nikitawootten-nist — APPROVED** [review]
Based on previous discussions we've had, this all looks great!

**Pull request merged**